extensions/pyRevitTools.extension/pyRevit.tab/Analysis.panel/ColorSplasher.pushbutton/script.py

```diff
@@ -862,15 +838,16 @@ def Button2Click(self, sender, e):
         self.reset_ev.Raise()
     
     def Button3Click(self, sender, e):
-        #Reassign Random colors
+        # Reassign Random colors
         try:
             checkindex = self._listBox1.CheckedIndices 
             for indx in checkindex:
                 self._listBox1.SetItemChecked(indx, False)
                 self._listBox1.SetItemChecked(indx, True)
         except:
-            pass        
-    
+            pass
+
+
```

Comment:
this change here is not needed, between class methods there should be only one white line